falcon/api.py

```diff
@@ -269,7 +269,8 @@ def __call__(self, env, start_response):  # noqa: C901
                     # next-hop child resource. In that case, the object
                     # being asked to dispatch to its child will raise an
                     # HTTP exception signalling the problem, e.g. a 404.
-                    responder, params, resource, req.uri_template = self._get_responder(req)
+                    responder, params, resource, req.uri_template = self._get_responder(
+                        req)
```

Comment:
Not a big deal, but do we need additional space for `self._get_responder`?

Reply:
You mean, move it to another line because it's past 80 characters? Or, are you asking if this change shouldn't have occurred?

I didn't intentionally make this change here, I think it must have been my auto-pep8. It looks like Falcon's pep8-checker rules allow 99-character lines, so probably I should revert this newline change. I guess my tool didn't pick up the config.

Reply:
Yeah, falcon allows 99-characters lines, [flake8 config](https://github.com/falconry/falcon/blob/master/tox.ini#L129).

In that case, I think would be better to revert that newline change